scripts/all-tests.sh

```diff
@@ -80,6 +80,7 @@ case "$TEST_SUITE" in
         # Run quality task. Pass in the 'fail-under' percentage to diff-quality
         paver run_quality -p 100
 
+        paver run_complexity > code_complexity.log || echo "Unable to calculate code complexity. Ignoring error."
```

Comment:
This needs to either get picked up as a build artifact or displayed on the console, otherwise you don't ever see the result in an actual build. 
E.g. the [quality subset](https://build.testeng.edx.org/job/edx-platform-test-subset/5359/) for the last commit on the branch.